### PR TITLE
EnergyOrigin update CD workflow 

### PR DESCRIPTION
**Pull request by MartinSchmidt**

### CODE REVIEW

File: .github/actions/get-semver/action.yaml [hunk 1]
```diff
@@ -12,14 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Get chart semver
+name: Get semver plus pr
 description: |
-  Get semver from the chart + pr if pr.
+  Get semver from a path in a yaml file and if PR then append extra values.
```
Comment (CodeReaper) [inline .github/actions/get-semver/action.yaml:17]: There is no `if` statements nor is there any appending happening.

File: .github/workflows/version-increment-check.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+      - name: Get current tag
+        id: current_tag
+        uses: Energinet-Datahub/.github/.github/actions/yaml-get@main
+        with: 
+          yaml_file: ${{ inputs.yaml_file }}
+          yaml_path: ${{ inputs.yaml_path }}
+
+      - name: Check versions
+        if: ${{ steps.current_tag.outputs.result != steps.semvers.outputs.patch && steps.current_tag.outputs.result != steps.semvers.outputs.minor && steps.current_tag.outputs.result != steps.semvers.outputs.major }}
```
Comment (CodeReaper) [inline .github/workflows/version-increment-check.yaml:65]: Comparing patch, minor and major against `steps.current_tag.outputs.result` seems incorrect.

**Review by Chrisbh — APPROVED** [review]

**Review by CodeReaper — CHANGES_REQUESTED** [review]

File: .github/actions/get-semver/action.yaml (hunk 1 above)
Comment (MartinSchmidt) [reply .github/actions/get-semver/action.yaml:17]: if exists in line 57 in bash.

File: .github/workflows/version-increment-check.yaml (hunk 2 above)
Comment (MartinSchmidt) [reply .github/workflows/version-increment-check.yaml:65]: steps.current_tag.outputs.result is the output value from the yaml-get that returns the current value from the yaml file.

**Review by CodeReaper — APPROVED** [review]

**Review by Renetnielsen — APPROVED** [review]

**Review by Renetnielsen — APPROVED** [review]

**Pull request merged**